Feature/venus adapter

0xmainnov opened the pull request:
- Add VenusMock contract for venus unit-tests & integration tests
- Add VenusLendAdapter contract

daiki-iu reviewed (changes requested):
@0xmainnov  please move test files into the folders I created separately for adapters.

0xmainnov commented:
@daiki-iu 
Just moved the tests by following current test folder architecture

daiki-iu commented on v1-contract/contracts/adapters/venus/venus-adapter-mock.sol:
```diff
@@ -9,7 +9,7 @@ import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 
 import "./interface/VBep20Interface.sol";
```
@0xmainnov  we don't need mock contract any more

daiki-iu reviewed (changes requested):
@0xmainnov  due to the update of overall workflow, please update unit test and integration test with new workflow.
- ybnft
- investor
- adapterManager
- venusLendAdapter

0xmainnov commented:
@daiki-iu 
I updated the unit-tests and also integration tests by followings
 - interaction tests with investor, adapter manager, yfNFT contracts
 - added some fixes of investor contract
 - add venus workflow including mint & redeem via investor contract

daiki-iu reviewed (approved):
@0xmainnov  LGTM!
